File: Iniciante/1071 - Soma de Impares Consecutivos I/Python/1071.py
def main():
    x = int(input())
    y = int(input())
    sum_ = 0
    end = max(x, y)
    start = min(x, y) + 1

    if start % 2 == 0:
        start += 1

    for i in range(start, end, 2):
        sum_ += i

    print(sum_)


if __name__ == "__main__":
    main()

File: Iniciante/1071 - Soma de Impares Consecutivos I/JavaScript/1071.js
function main() {
  const input = require("fs").readFileSync(0, "utf8").trim().split(/\s+/);
  const x = Number(input[0]);
  const y = Number(input[1]);
  let sum = 0;
  let end = Math.max(x, y);
  let start = Math.min(x, y) + 1;

  if (start % 2 === 0) {
    start += 1;
  }

  for (let i = start; i < end; i += 2) {
    sum += i;
  }

  console.log(sum);
}

main();
